functions/notify_slack.py

```diff
@@ -72,15 +72,19 @@ def notify_slack(payload):
     return result
 
 
-def get_guardduty_event(bucket, key):
```

Comment:
As it's fixing a bug, I guess it would be nice to see a test for this. However, I understand there isn't any existing tests so appreciate that you may not want to invest that amount of time in such a small change. I'll leave that decision up to you.

Reply:
Thanks for the input, having tests would be beneficial I was just concerned of the time investment. I will merge this then timebox making the tests.